mobile-academy/src/main/resources/smsTemplate.json

```diff
@@ -0,0 +1,18 @@
+{
+  "outboundSMSMessageRequest": {
+    "address": [
+      "tel: <phoneNumber>"
+    ],
+    "senderAddress": "tel: <senderId>",
+    "outboundSMSTextMessage": {
+      "message": "<messageContent>"
+    },
+    "clientCorrelator": "<correlationId>",
+    "receiptRequest": {
+      "notifyURL": "<notificationUrl>",
+      "callbackData": "$(callbackData)"
```

Comment:
The ones in <> are the ones that we replace. the $() is part of the template that IMI provides according to some GSMA spec that we ignore.